todos_impares_100.rb

```diff
@@ -0,0 +1,5 @@
+numbers = [*1..100]
+
+numbers.each_with_index do |number, position|
+    puts "#{numbers[position]}" if numbers[position].odd? == true
+end
```

Comment:
```suggestion
numbers.each do |number|
    puts "#{number}" if numbers.odd? == true
end
```

A ideia aqui também seria utilizarmos o `%` para criar a lógica do ímpar ou par, fazermos nossa própria implementação.